SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -305,3 +309,20 @@ void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel,
 		GlobalStateManager->HandleActorBasedOnLoadBalancer(Actor);
 	}
 }
+
+TSharedPtr<SpatialGDK::SpatialEventTracer> USpatialGameInstance::CreateEventTracer(const FString& WorkerId)
```

Comment:
This was moved from connection manager and an addition of storing `SpatialEventTracer` and returning it if already created

Reply:
Nit: We use the term WorkerName and WorkerId. WHAT IS IT DANNY!?

Reply:
Ahh I guess WorkerName is native and WorkerId is spatial :(